VideoLocker/src/test/java/org/edx/mobile/view/CourseUnitNavigationActivityTest.java

```diff
@@ -397,6 +395,7 @@ public void testUnitFragmentCreation() {
         CourseUnitFragment.HasComponent hasComponent = Mockito.mock(CourseUnitFragment.HasComponent.class);
 
         List<CourseComponent> unitList = new ArrayList<>();
+        List<Class> classesList = new ArrayList<>();
```

Comment:
You should also set the generic type (`List<Class<? extends CourseUnitFragment>>`).